sdks/python/README.md

```diff
@@ -255,215 +256,212 @@ Some interesting classes to navigate to:
 * combiners, in file
 [`google/cloud/dataflow/transforms/combiners.py`](http://localhost:8888/google.cloud.dataflow.transforms.combiners.html)
 
-## Some Simple Examples
+## Simple Examples
 
-### Hello world
+### Basic pipeline
 
-Create a transform from an iterable and use the pipe operator to chain
-transforms:
+A basic pipeline creates a `PCollection` from an iterable
```

Comment:
I think this needs to be somehow reworded.

Pipeline does not create a pcollection. Create transform create a pcollection from an iterable. It is a pipeline where that create transform is chained to other transforms with a pipe operator.

(You may leave changing this to another PR)